packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -316,6 +324,17 @@ export default class QuanticCaseClassification extends LightningElement {
       });
   }
 
+  /**
+   * Whether we want to fetch new case classifications or document suggestions when we update the field.
+   * @returns {Object}
```

Comment:
It's nice to be precise with the types when we can.
```suggestion
   * @returns {{caseClassifications: boolean, documentSuggestions: boolean}}
```